contracts/Rivendale.sol

```diff
@@ -111,19 +126,25 @@ contract Rivendale {
         );
         require(!voteBreakdown[_id].executed, "vote has already been settled");
         if (voteBreakdown[_id].tally > 500) {
+            // Should we allow votes with value?

```

Comment:
I don't think so...the value should come from the fellowship contract (slashing) not rivendale right?